src/LinearSolvers/cholmod.jl

```diff
@@ -5,7 +5,7 @@ end
 mutable struct CHOLMODSolver{T} <: AbstractLinearSolver{T}
     inner::CHOLMOD.Factor{Float64}
     tril::SparseMatrixCSC{T,Int32}
-    full::SparseMatrixCSC{Float64,Int32}
+    full::SparseMatrixCSC{Float64,Int}
```

Comment:
Just ouf of curiosity, why do we have to use `Int` there?

Reply:
For `VERSION <= v"1.9"`, cholmod throws an error if the argument is `full::SparseMatrixCSC{Float64,Int32}
`

Reply:
ah, good catch